src/database/migrations/2019_01_01_000000_create_cart_taggables_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateCartTaggablesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('taggables', function (Blueprint $table) {
```

Comment:
@AlexeiaW we just need to use the cart namespace here.

Reply:
Ok, changed the table name to cart_taggables